test/instrumentation/modules/@aws-sdk/fixtures/use-client-sns.mjs

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright Elasticsearch B.V. and other contributors where applicable.
+ * Licensed under the BSD 2-Clause License; you may not use this file except in
+ * compliance with the BSD 2-Clause License.
+ */
+
+'use strict';
+
+// A small subset of "./use-client-sns.js". Mainly this is to test that
+// instrumentation of @aws-sdk/client-sns in an ES module works.
+// See "./use-client-sns.js" for more details.
+//
+// Usage:
+//    node --experimental-loader=./loader.mjs --require=./start.js test/instrumentation/modules/@aws-sdk/fixtures/use-client-sns.mjs
+
+import apm from '../../../../../index.js'; // 'elastic-apm-node'
+import { SNSClient, PublishCommand } from '@aws-sdk/client-sns';
+
+import assert from 'assert';
+
+// ---- support functions
+
+async function useClientSNS(snsClient) {
+  const command = new PublishCommand({
+    Message: 'message to be sent',
+    PhoneNumber: '+34555555555',
```

Comment:
What *is* this phone number? A reserved (?) "555" code in Spain, I guess. Do you know if 555-5555 *is* a reserved test number?